src/client/pages/Events/EventsView.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import { withStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+import { Typography } from '@material-ui/core';
+
+import EventCard from './EventCard.jsx'
+
+function renderEvents(events) {
+  if (events.length === 0 || !events) {
+    console.log('EMPTY BOOOHOOO');
+    return (
+      <EventCard
+        month=''
+        day=''
+        weekday=''
+        name={'NOTHING\'S HAPPENING'}
+        timeloc=''
+        content='Please check back later'
+      />
+    );
+  }
+
+  const eventsCards = events.map((event, idx) => {
+    const month = event[0]
```

Comment:
I'm confused as to what this is.

Reply:
I was hoping to make the EventsView render a special EventCard whenever there is no events. Like the one below.
![special_no_events_card](https://user-images.githubusercontent.com/41027427/50395407-59040500-0729-11e9-9e60-b6adc8321526.PNG)